fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/scalar/FoldConstantsRule.java

```diff
@@ -191,7 +191,7 @@ public ScalarOperator visitLikePredicateOperator(LikePredicateOperator predicate
         return predicate;
     }
 
-    private boolean notAllConstant(List<ScalarOperator> operators) {
+    private boolean notAllConstantRef(List<ScalarOperator> operators) {
```

Comment:
Why rename this? constant no ref meaning.

Reply:
It corresponds to ScalarOperator::isConstantRef, we need use ScalarOperator::isConstant in some case later.

Reply:
I think `ScalarOperator::isConstantRef` also should rename to isConstantOperator, or directly check `instanceof ConstantOperator`.   The key point is constant no ref meaning.

Reply:
Done